src/js/components/ToggleGroup/StyledToggleGroup.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { Button } from '../Button';
+import { roundStyle, edgeStyle } from '../../utils';
+
+export const StyledButton = styled(Button)`
+  border-radius: 0;
+  ${(props) => roundStyle(props.round, true, props.theme)};
+  ${(props) =>
+    edgeStyle(
+      'padding',
+      props.icon !== undefined && props.theme.toggleGroup.iconOnly?.pad
```

Comment:
I think we want it only when it's iconOnly but not icon + label. Also, I wonder if it'd be cleaner though to have the logic in ToggleGroup itself so this is purely styles focused.

So in ToggleGroup where we're mapping and creating the Buttons:
```
const iconOnly = icon && !label;
let pad = theme.toggleGroup.button.pad;
if (iconOnly && theme.toggleGroup.button?.iconOnly?.pad) pad = theme.toggleGroup.button.iconOnly.pad;

...

<Button
... 
pad={pad}
/>
```

Reply:
done